src/main/scala/centaur/test/metadata/WorkflowMetadata.scala

```diff
@@ -27,16 +30,21 @@ case class WorkflowMetadata(value: Map[String, JsValue]) extends AnyVal {
       JsString, JsNumber and JsBoolean for comparison so this is a hacky way of handling that situation. It's
       entirely likely that it won't survive long term.
      */
+
+   def sanitizeUUID(jsValue: JsValue): String = if (jsValue.toString.contains("UUID")) { jsValue.toString.replace("UUID", workflowID) }
```

Comment:
Since this is only used for expected, you could just do
```
lazy val expectedString = expected.toString
lazy val sanitisedExpectedUUID = if (expectedString.contains("UUID")) expectedString.replace("UUID", workflowId) } else expectedString
```